groovy/groovy.editor/src/org/netbeans/modules/groovy/editor/api/GroovyIndexer.java

```diff
@@ -321,6 +323,9 @@ private void indexClass(ASTClass element, IndexDocument document) {
             document.addPair(FQN_NAME, element.getFqn(), true, true);
             document.addPair(CLASS_NAME, name, true, true);
             document.addPair(CASE_INSENSITIVE_CLASS_NAME, name.toLowerCase(), true, true);
+            StringBuilder sb = new StringBuilder();
```

Comment:
Q: `addPair(CLASS_NAME,...)` is called multiple times (toplevel classes, inner classes), but [it is read as a single value here](https://github.com/apache/netbeans/pull/4091/files#diff-d59a38cf1daed90d439de7eb0be4bcea950ef015a4f2321d2cfdd9a50b90a104R607) -- will this work OK with fields of inner classes ? If not I'd suggest to file a bug for Groovy & fix it in a later PR.

Reply:
I'm not sure what you mean, the link shows me the whole diff.  But inner classes, even with the same name as outer class should work and work.